gpflow/models/svgp.py

```diff
@@ -101,7 +100,14 @@ def build_prior_KL(self):
             K = None
         else:
             K = self.feature.Kuu(self.kern, jitter=settings.numerics.jitter_level)
-        return kullback_leiblers.gauss_kl(self.q_mu, self.q_sqrt, K)
+        M = tf.shape(self.q_mu)[0]
+
+        # M x L x R -> M x RL
+        # TODO: Extra transpose gives axes same order for q_mu as q_sqrt. Not necessary for valid KL. Keep?
```

Comment:
transposes aren't free... what's the advantage of this then?

Reply:
Code readability and that it's easier to see that the code is correct. Otherwise we're relying on slightly undefined behaviour of the KL divergence function. (i.e. that means and covariances don't interact in the whitened case). Actually, the more I think about it, the more I think that the extra transpose is necessary.

Reply:
Do you mean some implicit broadcasting in the KL divergence implementation? or what?